src/sql/parts/jobManagement/views/jobsView.component.ts

```diff
@@ -357,4 +361,8 @@ export class JobsViewComponent implements AfterContentChecked {
 			});
 		}
 	}
+
+	private showLoadingIcon(): boolean {
+		return this._showProgressWheel ? !this._isCloud : this._isCloud;
```

Comment:
I thought the extension shouldn't be available for Azure DB?  Should the fix be to prevent the extension from even showing in a context where it shouldn't be?

Reply:
I just did this to be sure, but this problem will never arise. Azure DB would never be shown due to this change - https://github.com/Microsoft/sqlopsstudio/pull/1233.
This change also fixes when there are no jobs available in a non-Azure DB.

Reply:
@abist I think there may be an issue where the `when` setting only controls initial visibility, but the extension tab can be manually edited.  Assuming that's still an issue we should fix the dashboard such that the `Add Extension` dialog also respects the `when` criteria.